bin/logstash.lib.sh

```diff
@@ -140,7 +132,9 @@ setup_ruby() {
 jruby_opts() {
   printf "%s" "--1.9"
   for i in $JAVA_OPTS ; do
-    printf "%s" " -J$i"
+    if [ "x$i" != "x" ] && [ "x$i" != "x " ]; then
```

Comment:
you can check if '$i' is empty by doing `if [ -z "$i" ]`. 

Also, when would $i be a space?